src/assets/lang/zh.json

```diff
@@ -175,8 +175,8 @@
       "title": "谢谢",
       "noSymptomsFeelingWell": "**感谢您今天的观察日志。我们很高兴您感觉身体状况不错！**\n\n即便您感觉状态不错，明日也还请继续告诉我们您的情况。",
       "riskGroup": "由于您的年纪，我们想请您格外小心保护自己免于感染COVID-19。\n\n我们知道这段时期不太好过。如果您需要关于如何居家防疫的支持或建议，请点击下面了解更多信息。",
-      "noSymptomsNotFeelingWell": "**感谢您今天的观察日志。得知您身体不适，我们深感抱歉。**\n\n幸运的是，您并未报告许多症状。如果您继续感到身体不适或出现新的症状，请告知您的医护人员或拨打纽约州健康热线[1-888-364-3065]（电话：12345）。\n\n如果您想接受检测，请联系您的医护人员或在公共检测点预约检测时间。\n寻找您附近的站点。\n\n请别忘了明天继续告诉我们您的情况。",
-      "coronavirus1": "**得知您身体不适，我们深感抱歉。不幸的是，您今天报告的症状可能是COVID-19症状。**\n\n如果您的症状轻微，请记住大多数人可以无需医疗护理即可在家中自愈。\n\n然而，我们建议任何出现COVID-19症状的人遵循以下步骤来保护他们自己和其他人的身体健康：\n\n**请待在家中。** 避免与其他人接触（包括您的家人）以免他们生病。\n\n**联系医护人员。**打电话给您的医生或拨打纽约州健康热线[1-888-364-3065]（电话：1234）。\n\n**接受检测。**咨询您的医护人员或在公共检测点预约检测时间。\n\n[寻找您附近的站点。](https://coronavirus.health.ny.gov/find-test-site-near-you)",
+      "noSymptomsNotFeelingWell": "**感谢您今天的观察日志。得知您身体不适，我们深感抱歉。**\n\n幸运的是，您并未报告许多症状。如果您继续感到身体不适或出现新的症状，请告知您的医护人员或拨打纽约州健康热线[1-888-364-3065](tel:18883643065)。\n\n如果您想接受检测，请联系您的医护人员或在公共检测点预约检测时间。\n寻找您附近的站点。\n\n请别忘了明天继续告诉我们您的情况。",
+      "coronavirus1": "**得知您身体不适，我们深感抱歉。不幸的是，您今天报告的症状可能是COVID-19症状。**\n\n如果您的症状轻微，请记住大多数人可以无需医疗护理即可在家中自愈。\n\n然而，我们建议任何出现COVID-19症状的人遵循以下步骤来保护他们自己和其他人的身体健康：\n\n**请待在家中。** 避免与其他人接触（包括您的家人）以免他们生病。\n\n**联系医护人员。**打电话给您的医生或拨打纽约州健康热线[1-888-364-3065](tel:18883643065)。\n\n**接受检测。**咨询您的医护人员或在公共检测点预约检测时间。\n\n[寻找您附近的站点。](https://coronavirus.health.ny.gov/find-test-site-near-you)",
```

Comment:
We have to admire the thoroughness of the translators here. As well as translating `tel`, they replaced the `(` and `)` with the wider Chinese variants.

We should also ask them not to 🙂 

(we should prob check the other Markdown links too)